### ISSUE TITLE
[Java] Add queues and stacks

### ISSUE THREAD

**Issue by geekygirlsarah** [codethesaurus/codethesaur.us#542]
# Description

Queues and stacks for Java aren't presently on Code Thesaurus. This issue is to add that concept to the system.

# Directions

* On the home page, on the right side (the reference sheet side), choose the language and version, and then the concept.
* The page says the data doesn't exist, but clicking the arrow for the template will provide the template and file for you to use.
* Save that file in `/web/thesauruses/[language]/[version]/queues_stacks.json`.
* Fill out all of the `"code"` sections.
* Add in `"not-implemented": "true"` if the concept doesn't exist in the language. You can then delete the `"code"` line.
* Add in `"comment":` if you wish to explain something in the code
* Delete the `"name":` fields before your PR
* If you want to validate your file, run `python manage.py validatelanginfofiles`
* Save, commit your code, push to your fork, then make a PR.
* Watch the checks to make sure they all pass. If they don't, click on "Details" and it should tell you what to modify.

# Claiming This Issue

Please leave a comment below so I can assign you to it. This is to prevent multiple people from working on the same issue. 

Check out [CONTRIBUTING.md](https://github.com/codethesaurus/codethesaur.us/blob/main/CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on how Code Thesaurus works and how to add to the thesauruses.

# Need Help?

Leave a comment on here with your question, or feel free to ask [@geekygirlsarah](https://twitter.com/geekygirlsarah) on Twitter. You can also email coreteam@codethesaur.us too.

# Thanks!

We really value and appreciate the additions people make to Code Thesaurus, no matter how large or small. So thank you for taking the time to work on this!

**Comment by ujinikalyan** [comment]
hey can you assign this form me? @geekygirlsarah

**Comment by geekygirlsarah** [comment]
Yeah! Be sure to look over the docs and let me know if you have any questions!

**Comment by ujinikalyan** [comment]
hey @geekygirlsarah I need add the queues_and_stacks.json files in /web/thesauruses/java/15 thats it? Also how can I test run this?

**Comment by geekygirlsarah** [comment]
Look at the Install/running docs at https://docs.codethesaur.us/install/ to see how to run it locally.

And yeah, basically add that, though if there's other versions, it'd be nice to add those too. (LIke Java has 11, 15, and 17. I imagine they're all pretty similar as I'm guessing queues and stacks haven't changed much through the years)

**Comment by ujinikalyan** [comment]
hey @geekygirlsarah sorry for the delay in the work I am almost done with the issue give me a couple of days and I'll send pr on the issue. I am asking for extra time as it is hard for me to continue my hobby and learn more about open-source contribution with my placements going on as I am a final year undergrad student. Sorry for the delay.

**Comment by geekygirlsarah** [comment]
That's fine! No rush. I have a bot that will remove someone from an issue at 2.5 weeks (though it warns first). So if the bot shows up, leave a comment and it won't take you off.

Though if you're doing Hacktoberfest, the PR needs to be made by Oct 31.
